### DISCUSSION TITLE
RasPi Wifi and Nexmon

### DISCUSSION THREAD

**Discussion by IGNNE** [svpcom/wfb-ng#133] [Questions] [answered]
Hello,

I am not sure if this is the right place, but I want to improve my link by using the internal Wifi chip of my RasPi 3A in addition to an external card. The external card works just fine, but I can't seem to transmit anything with the internal one.

I am using https://github.com/rcatolino/nexmon for the RasPi, together with Kernel 5.4. With this, I can successfully watch beacons and control messages from regular Wifi in tcpdump, but WFB never shows up anywhere. Is there a more low-level way of "just creating some random traffic" or something? Has anyone else tried this? I found Issue #12, but that was a long time ago.

Thanks

**Comment by svpcom** [comment, marked as answer]
https://www.aircrack-ng.org/doku.php?id=aireplay-ng

**Reply by IGNNE** [reply]
Damn, I completely forgot about aircrack, thank you!

I found the issue, injection simply does not work with that branch, but it works with the original nexmon and the bcm43455c0/7_45_206 firmware/driver (as suggested by https://github.com/seemoo-lab/nexmon/pull/452#issuecomment-750011728). 

I am going to try WFB and see if it works now.

**Reply by IGNNE** [reply]
Alright, I tried it, but I ran into this bug: https://github.com/seemoo-lab/nexmon/issues/335

I think that for now the internal chip is not working well enough for WFB.

**Comment by Joshy-2010** [comment]
I am kinda curious, did you actually get it to work @IGNNE, even if only for a short time? 

Because I really want to try it out again, idealy with a Pi Zero 2W, but currently where I live they are very expensive.Would be a great little FPV setup for under 15gramms!

I hope these bugs you talked about might be fixed by now. I'll report my progress back here, but any tips in advance would be greatly appreciated, if you see this post.
